docs/tests/integration/test_remoted/test_manager_messages/test_manager_ack.md

```diff
@@ -0,0 +1,52 @@
+# Test manager ACK
+
+## Overview
+
+These tests will check if the manager sends the ACK message after receiving the `start-up` message from agent.
+
+## Objective
+
+The objective is to check that the manager sends the ACK message using the different protocols.
+
+## General info
+
+|Tier | Number of tests | Time spent |
+|:--:|:--:|:--:|
+| 0 | 4 | 1m 15s |
+
+## Expected behavior
+
+Success if the agent receives the ACK message from the manager after sending the `start-up` message. Failure otherwise.
+
+## Testing
+
+The testing is based on configuring the manager to receive messages via `TCP`, `UDP`, `TCP-UDP` and `UDP-TCP`.
+
+First, the simulated agent will send the `start-up` message to the manager, and then, the agent will save all the
+incoming messages from the agent in a buffer.
```

Comment:
It would be great to add more details about the `start-up` message, its format, etc. As you did right below with the `ack`.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/1109/commits/b7f883a66179468ce27081052ce5e2d4dbb0a617